### PR TITLE
I deleted the ui package

### PR DESCRIPTION
**Pull request by AdamKane44**
Please review these changes!

### CODE REVIEW

File: src/ui/MenuSetup.java [hunk 1]
```diff
@@ -1,156 +0,0 @@
...
-        System.out.println("3. Remove Employee");
-        System.out.println("4. List Projects");
-        System.out.println("5. Add Project");
-        System.out.println("6. Delete Project");
-        System.out.println("7. Exit");
-    }
-
-    //use case statement to dictate what is done with user selection
-    public static void handleUserSelection(int selection)
-    {
```
Comment (ElliotB-96) [inline src/ui/MenuSetup.java]: this is terrible

**Comment by AdamKane44** [comment]
I know 👎 👎

File: src/ui/MenuSetup.java [hunk 2]
```diff
@@ -1,156 +0,0 @@
...
-import employee_stuff.ProjectController;
-
-import java.util.ArrayList;
-
-import java.util.Scanner;
-
-
-public class MenuSetup {
-    // boolean for checking if menu is still printing
-    static boolean keepPrinting = true;
```
Comment (Thomas-Franklin) [inline src/ui/MenuSetup.java]: Plz, thankfully this was deleted.

File: src/ui/MenuSetup.java [hunk 3]
```diff
@@ -1,156 +0,0 @@
-package ui;
-
-import database.DatabaseConnector;
-import employee_stuff.EmployeeController;
-import employee_stuff.Employee;
-import employee_stuff.Project;
-import employee_stuff.ProjectController;
-
-import java.util.ArrayList;
```
Comment (Thomas-Franklin) [inline src/ui/MenuSetup.java]: Use real Arrays, find out the length.

**Pull request closed without merging**